worker/evCheckerWorker/evCheckerWorker.go

```diff
@@ -0,0 +1,100 @@
+package evCheckerWorker
+
+import (
+	"bytes"
+	"encoding/pem"
+	"fmt"
+	"github.com/mozilla/tls-observatory/logger"
+	"github.com/mozilla/tls-observatory/worker"
+	"io/ioutil"
+	"os/exec"
+	"reflect"
+)
+
+var workerName = "evCheckerWorker"
+var workerDesc = `Provides insights for EV certificates`
+
+var log = logger.GetLogger()
+
+func init() {
+	worker.RegisterWorker(workerName, worker.Info{Runner: new(evWorker), Description: workerDesc})
+}
+
+type evWorker struct{}
+
+func (w evWorker) Run(in worker.Input, res chan worker.Result) {
+	scan, err := in.DBHandle.GetScanByID(in.Scanid)
+	if err != nil {
+		w.error("Could not get scan: %s", err, res)
+		return
+	}
+
+	params := reflect.ValueOf(in.Params)
+	if params.Kind() != reflect.Map {
+		w.error("%s", fmt.Errorf("Invalid parameters passed to evWorker: %s", params), res)
+		return
+	}
+	var oid string
+	for _, k := range params.MapKeys() {
+		if k.Interface() == "oid" {
+			var ok bool
+			oid, ok = params.MapIndex(k).Interface().(string)
+			if !ok {
+				w.error("%s", fmt.Errorf("Could not cast oid to string"), res)
+			}
+		}
+	}
```

Comment:
Rather than reflecting by yourself, an inelegant solution that I used a couple times is to json marshal the interface, then unmarshal it into a local struct, like this:
```go
type myParamsStruct struct {
    OID string `json:"oid"`
}

paramsData, err := json.Marshal(in.Params)
var myparams myParamsStruct
err = json.Unmarshal(params.Data, &myparams)
```
The benefit of this approach is its generic to all workers, as opposed to writing custom reflection code everywhere. As such, it can be moved into the `workers` package instead of living here. For an example, take a look at https://github.com/mozilla-services/userplex/blob/master/modules/modules.go#L72-L80